### PR TITLE
Implemented the inventory table as a react table and added styling to DataTable.css

### PR DESCRIPTION
**Pull request by katelyn-schmidt**
(solves issues #19 and #40 )
1. Updated inventory.js and DataTable.js to use react table by referencing the existing react table for the orders table. 
2. Added styling to DataTable.css including:
- changed fonts for both list text and title text
- changed font sizes to be bigger
- changed the column headers to be bolded
- made the column headers background green to match Ethical colors
- spaced the lines in the table to be less crowded and easier to read
- made the striped highlighting background lighter to make words easier to read
- changed the hover highlight color to yellow to match Ethical colors

### CODE REVIEW

File: src/components/DataTable.css [hunk 1]
```diff
@@ -1,36 +1,51 @@
...
+.Inventory {
+  font-family: 'Paytone One', sans-serif;
+  font-size: x-large;
+  color: rgb(80, 80, 80);
+}
+
+.Orders {
+  font-family: 'Paytone One', sans-serif;
+  font-size: x-large;
+  color: rgb(80, 80, 80);
```
Comment (trev4ev) [inline src/components/DataTable.css]: These can be combined (search how to apply style to multiple classes css)

File: src/components/DataTable.js [hunk 2]
```diff
@@ -1,60 +1,60 @@
...
+              desc: false
+            }
+          ]}
+          filterable={true}
+          defaultFiltered={[
+            {
+              id: "Pick Up Date",
+              value: "12/31/2025"
+            }
+          ]}
```
Comment (trev4ev) [inline src/components/DataTable.js]: Pick up date does not exist for this table so you should be using another column

File: src/components/Inventory.js [hunk 3]
```diff
@@ -16,14 +17,18 @@ class Inventory extends React.Component {
...
+      };
+    });
     return (
-      <div>
-        <div className="Inventory">
-          <h1 className="header">Inventory</h1>
-          <DataTable headings={headings} rows={this.state.api_rows} />
-        </div>
+      <div className="Inventory">
+        <h1 className="header"> INVENTORY </h1>
```
Comment (trev4ev) [inline src/components/Inventory.js]: Make this change to orders headers if you want all uppercase. Also please add a header to Announcements just so we are being consistent.

File: src/components/DataTable.js [hunk 4]
```diff
@@ -1,60 +1,60 @@
...
+          data={this.props.arrayOfObjects}
+          columns={[
+            {
+              Header: "",
+              columns: []
+            },
+            {
+              Header: "Item",
+              accessor: "Item",
+              filterable: false
```
Comment (trev4ev) [inline src/components/DataTable.js]: can you add filtering (aka sorting) by item name. see orders table for how this is done

**Review by trev4ev — CHANGES_REQUESTED** [review]
please take a look!

**Review by abaruah117 — DISMISSED** [review]
Looks good, same as trevor's comments but once you resolve those should be good to go!

**Comment by katelyn-schmidt** [comment]
Made changes to fix PR comments, please check it out! @trev4ev @abaruah117

**Review by trev4ev — APPROVED** [review]
lgtm

**Review by abaruah117 — APPROVED** [review]
lgtm

**Pull request merged**